.github/workflows/ci.yml

```diff
@@ -118,6 +121,74 @@ jobs:
+          ddev drush cache:rebuild -y
+
+          # identify set of configuration changes before running updatedb and store it in a text file.
+          # awk NF is used to filter blank lines.
+          ddev drush config:status --fields name --format list | awk NF > ${{ runner.temp }}/initial_config.txt
+
+          # run updates and build cache
+          ddev drush updatedb --yes
+          ddev drush cache:rebuild --yes
+
+          # identify set of configuration changes after running updatedb and store it in a text file.
+          ddev drush config:status --fields name --format list | awk NF > ${{ runner.temp }}/updated_config.txt
+
+          # sort the lines within files as comm requires sorted files
+          sort ${{ runner.temp }}/initial_config.txt -o ${{ runner.temp }}/sorted_initial_config.txt
+          sort ${{ runner.temp }}/updated_config.txt -o ${{ runner.temp }}/sorted_updated_config.txt
+
+          # This will compare above sorted files and write unique lines from second list which is not present in first list.
+          comm -13 ${{ runner.temp }}/sorted_initial_config.txt ${{ runner.temp }}/sorted_updated_config.txt > ${{ runner.temp }}/mismatched_config.txt
+
+          # If mismatched_config is not empty that means we have missing config export. We store it in GITHUB_OUTPUT for next steps.
+          if [ -s ${{ runner.temp }}/mismatched_config.txt ]; then
+            echo "Config export has differences"
+            cat ${{ runner.temp }}/mismatched_config.txt
+            # We are using EOF-based output as mismatched_config.txt may contain multiple lines
+            status_output=$(cat ${{ runner.temp }}/mismatched_config.txt)
+            echo "status_output<<EOF" >> $GITHUB_OUTPUT
+            echo "$status_output" >> $GITHUB_OUTPUT
+            echo "EOF" >> $GITHUB_OUTPUT
+          else
+            echo "Config has no differences"
+            echo "status_output=No differences" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Add PR Comment if Validate Config fails
+        if: ${{ !contains(steps.validate_config.outputs.status_output, 'No differences') }}
+        uses: mshick/add-pr-comment@v2
+        with:
+          message: |
+            **Config export has differences**
+            ```
+            ${{ steps.validate_config.outputs.status_output }}
+            ```
+
+      # Update the comment only if it was added at first place
+      - name: Update the comment on success
+        if: ${{ contains(steps.validate_config.outputs.status_output, 'No differences') }}
+        uses: mshick/add-pr-comment@v2
+        with:
+          update-only: true
+          message: |
+            ```
+            ${{ steps.validate_config.outputs.status_output }}
+            ```
+
+      - name: Fail if Config Export is not clean
+        if: ${{ !contains(steps.validate_config.outputs.status_output, 'No differences') }}
+        run: |
+          echo "Config export has differences"
+          exit 1
```

Comment:
We discussed that there are some edge cases where this could get reported. Since we know that, let's not abandon our tests. If GitHub Actions has a way to report warning (I vaguely remember something like that), let's do that. In any case, let's not stop executing the tests.

Reply:
Ok sure, will implement this